extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -36,8 +39,115 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
   transition: opacity 0.2s cubic-bezier(0, 0, 0.2, 1) !important;
 }
 
-/** Make the dialog wrapper from amp-subscriptions round corner. */
+/** Override the dialog styles from amp-subscriptions. */
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
   border-radius: 16px 16px 0px 0px !important;
   font-family: Poppins, sans-serif !important;
+  font-weight: 400 !important;
+  background: white !important;
+  color: black !important;
+  overflow: hidden !important;
+}
+amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
+  width: 100% !important;
+  left: 0 !important;
+  margin-left: 0 !important;
+}
+@media(min-aspect-ratio: 3/4) {
+  amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
+    width: var(--i-amphtml-story-desktop-one-panel-width) !important;
+    left: 50% !important;
+    margin-left: calc(-0.5 * var(--i-amphtml-story-desktop-one-panel-width)) !important;
+  }
+}
+
+.i-amphtml-story-subscriptions-dialog-banner {
+  height: 44px !important;
+  background: #EDEDED !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-content {
+  margin: 28px 28px 0px !important;
+  text-align: center !important;
+  white-space: nowrap !important;
+  overflow: hidden !important;
+  text-overflow: ellipsis !important;
+}
+
+.i-amphtml-story-subscriptions-price {
+  font-weight: 400 !important;
+  font-size: 24px !important;
+  letter-spacing: -0.4px !important;
+  height: 41px !important;
+}
+
+.i-amphtml-story-subscriptions-title {
+  font-weight: 700 !important;
+  font-size: 20px !important;
+  letter-spacing: -0.25px !important;
+  height: 41px !important;
+}
+
+.i-amphtml-story-subscriptions-subtitle-first, .i-amphtml-story-subscriptions-subtitle-second {
+  font-size: 16px !important;
+  letter-spacing: -0.25px !important;
+  height: 22px !important;
+}
+
+.i-amphtml-story-subscriptions-subtitle-second {
+  margin-bottom: 39px !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-button, .i-amphtml-story-subscriptions-google-button {
+  margin: 0 auto 12px !important;
+  border: none !important;
+  width: 212px !important;
+  height: 40px !important;
+  border-radius: 51px !important;
+  display: flex !important;
+  justify-content: center !important;
+  align-items: center !important;
+  cursor: pointer !important;
+}
+.i-amphtml-story-subscriptions-publisher-button {
+  background: black !important;
+}
+.i-amphtml-story-subscriptions-google-button {
+  background: #F2F2F2 !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-logo, .i-amphtml-story-subscriptions-google-logo {
+  margin-right: 10px !important;
+  width: 32px !important;
+  height: 32px !important;
+  border-radius: 50% !important;
+  margin-left: -12px !important;
```

Comment:
it's good to not use negative values as a rule of thumb for CSS.
Negative values can make debugging CSS difficult.